README.md

```diff
@@ -33,14 +38,15 @@ In order to transform Unreal from a single server engine to a distributed model,
 ## Give us feedback
 We have released the GDK for Unreal this early in development because we want your feedback. Please come and talk to us about the software and the documentation via: [Discord](https://discordapp.com/channels/311273633307951114/339471548647866368) - [Forums](https://forums.improbable.io/) - [GitHub issues in this repository](https://github.com/spatialos/UnrealGDK/issues).
 
-## Where to get the GDK and starter projects
-The GDK and its starter projects are available on GitHub.
+## Where to get the GDK and example projects
```

Comment:
Should we say "related projects" instead? Some of these are not really example projects.